packages/react-instantsearch-dom-maps/src/__tests__/withGoogleMaps.tsx

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
 import Enzyme, { mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import withGoogleMaps, {
-  GoogleMapsContext,
-  WithGoogleMapsProps,
-} from '../withGoogleMaps';
+import withGoogleMaps, { WithGoogleMapsProps, } from '../withGoogleMaps';
 
 Enzyme.configure({ adapter: new Adapter() });
 
-describe('withGoogleMaps', () => {
+describe.skip('withGoogleMaps', () => {
```

Comment:
To be tested correctly, this will probably also need an inner component to which we pass the context as a prop.

Reply:
https://github.com/algolia/react-instantsearch/pull/2023#discussion_r256762319